src/probability_theory/martingale.lean

```diff
@@ -19,6 +19,10 @@ The definitions of filtration and adapted can be found in `probability_theory.st
 
 * `measure_theory.martingale f ℱ μ`: `f` is a martingale with respect to filtration `ℱ` and
   measure `μ`.
+* `measure_theory.supermartingale f ℱ μ`: `f` is a supermartingale with respect to
+  filtration `ℱ` and measure `μ`.
+* `measure_theory.submartingale f ℱ μ`: `f` is a submartingale with respect to filtration `ℱ` and
+  measure `μ`.
```

Comment:
Could you also add a description of what a supermartingale is? Near lines 13-16 for example.